src/iv/inputs.ts

```diff
@@ -42,6 +42,13 @@ export const value = decorator(Value, ($api: Value) => {
         let v: any;
         if (inputType === "text" || inputType === "number") {
             v = elt[VALUE];
+        } else if (inputType === "range") {
+            const value = elt[VALUE];
+            try {
+                v = parseInt(value);
```

Comment:
Note that `parseInt` does not raise an exception when the value is not a number! It returns `NaN`.
So what's the purpose of this try...catch block?

Reply:
Fixed.